projects/aws-containers/hello-eks-anywhere/helm/schema.json

```diff
@@ -4,11 +4,33 @@
   "title": "hello-eks-anywhere",
   "type": "object",
   "properties": {
+    "namespace": {
+      "type": "string",
+      "default": "default",
+      "description": "Namespace for package and resources."
+    },
```

Comment:
I think adding namespace here would be confusing because we aren't going to use it from the package `config` we'll use `targetNamespace`